src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/entities/search/SearchParameterKeyTest.java

```diff
@@ -32,4 +35,49 @@ void update_test_whenever_additions_to_assign_enum_are_made() {
         int assigneeEnumLength = SearchParameterKey.values().length;
         assertEquals(8, assigneeEnumLength);
     }
+
+
+    @ParameterizedTest
+    @CsvSource(
+        value = {
+            "LOCATION, location",
+            "USER, user",
+            "JURISDICTION, jurisdiction",
+            "STATE, state",
+            "TASK_ID, taskId",
+            "TASK_TYPE, taskType",
+            "CASE_ID, caseId",
+            "WORK_TYPE, work_type"
+        }
+    )
```

Comment:
While we support camel case for R1 properties I believe we should migrate these to snake case maybe in another ticket